certs/certs.go

```diff
@@ -10,7 +11,7 @@ import (
 	"github.com/deis/controller-sdk-go/api"
 )
 
-// List certs registered with the controller.
+// List lists certificates added to deis.
```

Comment:
Ah... I was not aware of that. If that's a well-established practice I didn't know about, then cool. Carry on. This was non-blocking anyway.